packages/cysync-core/src/hooks/closeDialogBox.ts

```diff
@@ -0,0 +1,9 @@
+import { DialogName, closeDialog, useAppDispatch } from '~/store';
+
+export const useCloseDialogBox = () => {
+  const dispatch = useAppDispatch();
+  const closeDialogBox = (dialog: DialogName) => {
+    dispatch(closeDialog(dialog));
+  };
+  return closeDialogBox;
+};
```

Comment:
I don't think we need this. The `onClose` function should be defined in the context itself. Check `AddAccount`

Reply:
done